### DISCUSSION TITLE
Change language of notifications

### DISCUSSION THREAD

**Discussion by hollosch** [lukevella/rallly#516] [Q&A] [answered]
Hi,
is it possible to change the notifications / notifications-emails into another language?

thx

**Comment by lukevella** [comment, marked as answer]
At the moment it’s not a trivial task to do this but I plan to refactor the code for sending Email notifications in the next couple of weeks so that will be a good time to work on making this possible.

**Reply by hollosch** [reply]
Thx, this would be very nice!
